src/tests/AudioProgress.test.js

```diff
@@ -8,11 +8,13 @@ import AudioProgress from 'components/AudioPlayer/AudioProgress';
 
 test('it renders', () => {
   const ref = React.createRef();
+  const mockFn = jest.fn();
```

Comment:
what's this? I am new to jest.

Reply:
it's a mock function you can use, it keeps track of how many times it was called. I'm using it here just because the component needs a function. I'm only testing the rendering here. 😄